test/testcne/tailqs_test.c

```diff
@@ -0,0 +1,157 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) <2019-2022>, Intel Corporation. All rights reserved.
```

Comment:
Is this ported from DPDK? I ask because the copyright is different than most of the other files that use something liiek

```
/* SPDX-License-Identifier: BSD-3-Clause
 * Copyright (c) 2019-2022 Intel Corporation
 */
```

It's a minor thing, but it makes it easier for scripts that need to parse these headers if they're all the same.